modules/core/src/main/java/org/apache/ignite/internal/processors/service/ServiceContextImpl.java

```diff
@@ -131,6 +137,11 @@ ExecutorService executor() {
         return exe;
     }
 
+    /** @return {@code True} if statistics is enabled for this service. {@code False} otherwise. */
+    boolean statisticsEnabled() {
```

Comment:
1. Why not `isStatisticsEnabled()`

Reply:
Because the internal packages. 'Getters and setters should be implemented without is/get/set prefixes in internal APIs.':  https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Gettersandsetters

Reply:
This is insonsistent, this class implements public interface. It has method implemented method `isCancelled()`.
It is weird to have different policies in one file.

Reply:
Fixed